cockatrice/src/tab_deck_editor.cpp

```diff
@@ -700,7 +704,11 @@ void TabDeckEditor::updateCardInfoRight(const QModelIndex &current, const QModel
 
 void TabDeckEditor::updateSearch(const QString &search)
 {
-    databaseDisplayModel->setCardName(search);
+    if (search.contains(QRegularExpression("[:<>=\"!]"))) {
```

Comment:
Wouldn't it be a tad more efficient to store the regex instead? afaik qt does some optimization for them.

Reply:
Agreed, having a const QRegex is better